test/powershell/Modules/Microsoft.PowerShell.Management/Get-ComputerInfo.Tests.ps1

```diff
@@ -341,11 +341,16 @@ public static extern int LCIDToLocaleName(uint localeID, System.Text.StringBuild
         $returnValue = $null
         try
         {
-            $returnValue = Get-CimClassPropVal Win32_DeviceGuard $propertyName -namespace 'root\Microsoft\Windows\DeviceGuard' -ErrorAction Stop
+            #Since we want to catch any exceptions here, we won't use the above Get-Cim* helper functions
+            $inst = Get-CimInstance Win32_DeviceGuard -namespace 'root\Microsoft\Windows\DeviceGuard' -ErrorAction Stop
+            $returnValue = $inst.$propertyName
         }
-        catch
+        catch [Microsoft.Management.Infrastructure.CimException]
         {
-            #swallow this
+            if ($_.Exception.ErrorData.CimInstanceProperties["error_code"].value -ne [System.Convert]::ToUInt32("0x8004100E", 16))
```

Comment:
Use the `ShouldBeErrorId` [pattern](https://github.com/PowerShell/PowerShell/search?utf8=%E2%9C%93&q=shouldbeerrorid)